src/modules/AieraChat/index.tsx

```diff
@@ -39,10 +41,12 @@ export function AieraChat(): ReactElement {
     const virtuosoRef = useRef<VirtuosoMessageListMethods<ChatMessage>>(null);
 
     // Set up Ably realtime client
-    const { createAblyRealtimeClient } = useAbly();
+    const { createAblyRealtimeClient, subscribeToChannel, unsubscribeFromChannel } = useAbly();
     const [clientReady, setClientReady] = useState(false);
     const initializingRef = useRef(false);
     const clientRef = useRef<Ably.Realtime | null>(null);
+    const [subscribedTitleChannel, setSubscribedTitleChannel] = useState<RealtimeChannel | undefined>(undefined);
+    const titleChannelName = useMemo(() => (chatId !== 'new' ? `${CHANNEL_PREFIX}:${chatId}:title` : ''), [chatId]);
```

Comment:
Can we move this to the use effect where it's used since it seems to only be used there? We also shouldn't need to memo this.